tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/TelemetryTests.cs

```diff
@@ -160,16 +174,21 @@ async Task RunTest()
                 agent.AssertIntegrationEnabled(IntegrationId.HttpMessageHandler);
                 agent.AssertConfiguration(ConfigTelemetryData.NativeTracerVersion, TracerConstants.ThreePartVersion);
                 AssertService(agent, "Samples.Telemetry", ServiceVersion);
+                AssertDependencies(agent, enableDependencies);
             }
         }
 
 #if NETCOREAPP3_1_OR_GREATER
-        [SkippableFact]
+        [SkippableTheory]
         [Trait("Category", "EndToEnd")]
         [Trait("RunOnWindows", "True")]
-        public async Task WhenUsingUdsAgent_UsesUdsTelemetry()
+        [InlineData(null)]
+        [InlineData(true)]
+        [InlineData(false)]
+        public async Task WhenUsingUdsAgent_UsesUdsTelemetry(bool? enableDependencies)
         {
             EnvironmentHelper.EnableUnixDomainSockets();
+            EnableDependencies(enableDependencies);
             using var agent = EnvironmentHelper.GetMockAgent(useTelemetry: true);
 
             int httpPort = TcpPortProvider.GetOpenPort();
```

Comment:
Don't you want to assert dependencies in that test?

Reply:
Yep, good catch